src/main/webapp/app/shared/layouts/profiles/profile.service.ts

```diff
@@ -73,11 +77,19 @@ export class ProfileService {
     }
 
     private mapAllowedOrionVersions(data: any, profileInfo: ProfileInfo) {
-        profileInfo.allowedMinimumOrionVersion = data['allowed-minimum-orion-version'];
+        if (data['allowed-minimum-orion-version']) {
+            profileInfo.allowedMinimumOrionVersion = data['allowed-minimum-orion-version'];
+        } else {
+            profileInfo.allowedMinimumOrionVersion = data['allowedMinimumOrionVersion'];
+        }
     }
 
     private mapTestServer(data: any, profileInfo: ProfileInfo) {
-        profileInfo.testServer = data['test-server'];
+        if (data['test-server']) {
+            profileInfo.testServer = data['test-server'];
+        } else {
+            profileInfo.testServer = data['testServer'];
```

Comment:
I don't really understand the else clause here? Why should the key change to camel case?

Reply:
Please have a look at my reply above to a similar issue.

Reply:
My fault. (See description or answer above)